src/flare/flr-dca-cooldown.rain

```diff
@@ -0,0 +1,76 @@
+tokens:
+  eusdt:
+    address: 0x96B41289D90444B8adD57e6F265DB5aE8651DF29
+    decimals: 6
+  wflr:
+    address: 0x1D80c49BbBCd1C0911346656B529DF9E5c2F783d
+    decimals: 18
+
+orders:
+  dca-buy:
+    inputs:
+        # EUSDT
+        - token: 0x96B41289D90444B8adD57e6F265DB5aE8651DF29
+          decimals: 6
+          vault-id: 0x01
+    outputs:
+        # WFLR
+        - token: 0x1D80c49BbBCd1C0911346656B529DF9E5c2F783d
+          decimals: 18
+          vault-id: 0x02
+scenarios:
+  default:
+    bindings:
+      flare-subparser: 0x4F4267c2084607A5e23660733Cb3a0Cf5840F0C7
+      cooldown-time: 300
+    scenarios:
+      sell-wflr:
+        bindings:
+          per-trade-amount: 100e18
+          bounty-amount: 1e15
+          ftso-base: "FLR"
+          ftso-quote: "USD"
+      buy-wflr:
+        bindings:
+          per-trade-amount: 100e18
+          bounty-amount: 1e15
+          ftso-base: "USD"
+          ftso-quote: "FLR"
+
+deployments:
+  sell-wflr:
+    scenario: default.sell-wflr
+    order: sell-wflr
+  buy-wflr:
+    scenario: default.buy-wflr
+    order: buy-wflr
+---
+/* This strategy will buy or sell WFLR based on the price of the FTSO pair FLR/USD */
+
+#flare-sub-parser !The flare subparser address
+
+#per-trade-amount !The amount that we'll buy or sell per trade
+#cooldown-time !The minimum amount of time that must elapse between trades
+#bounty-amount !The amount below the oracle price that we are willing to buy/sell at
```

Comment:
units are unclear here, looks like in the strat you're subtracing an absolute value from a ratio?